sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -331,22 +389,23 @@ def _download_file(from_url, to_path):
   def _is_remote_path(path):
     return path.find('://') != -1
 
-  def _stage_jar_packages(self, jar_packages, staging_location, temp_dir):
-    # type: (...) -> List[str]
+  @staticmethod
+  def _stage_jar_packages(jar_packages, temp_dir):
```

Comment:
Does this actually stage them, or just add them to the returned list? If not, the name and docstring should be updated.

Reply:
done.